storage/adapters/base_adapter.js

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+/* eslint-disable class-methods-use-this */
+
+const { EventEmitter } = require('events');
+const { ImplementationPendingError } = require('../errors');
+
+const EVENT_CONNECT = Symbol('ADAPTER_EVENT_CONNECT');
```

Comment:
why `Symbol` here? simple string might be more useful?

Reply:
Can you give an example where string would be more useful?

Reply:
maybe intended, but user need to use this const to listen to event, where in string, u can just define the same string to listen to the event.
I feel like it doesn't need to be that strict

Reply:
Even if it's a string, the best practice is to use the const instead of just defining the same string.
I personally like enforce using the same const.